contracts/AssetPool.sol

```diff
@@ -296,6 +324,80 @@ contract AssetPool is Ownable, IAssetPool {
         collateralToken.safeTransfer(recipient, amount);
     }
 
+    function beginWithdrawalDelayUpdate(uint256 _newWithdrawalDelay)
+        external
+        onlyOwner
+    {
+        newWithdrawalDelay = _newWithdrawalDelay;
+        withdrawalDelayChangeInitiated = block.timestamp;
+        emit WithdrawalDelayUpdateStarted(_newWithdrawalDelay, block.timestamp);
+    }
+
+    function finalizeWithdrawalDelayUpdate()
+        external
+        onlyOwner
+        onlyAfterWithdrawalGovernanceDelay(withdrawalDelayChangeInitiated)
+    {
+        withdrawalDelay = newWithdrawalDelay;
+        emit WithdrawalDelayUpdated(withdrawalDelay);
+        newWithdrawalDelay = 0;
+        withdrawalDelayChangeInitiated = 0;
+    }
+
+    function beginWithdrawalTimeoutUpdate(uint256 _newWithdrawalTimeout)
+        external
+        onlyOwner
+    {
+        newWithdrawalTimeout = _newWithdrawalTimeout;
+        withdrawalTimeoutChangeInitiated = block.timestamp;
+        emit WithdrawalTimeoutUpdateStarted(
+            _newWithdrawalTimeout,
+            block.timestamp
+        );
+    }
+
+    function finalizeWithdrawalTimeoutUpdate()
+        external
+        onlyOwner
+        onlyAfterWithdrawalGovernanceDelay(withdrawalTimeoutChangeInitiated)
+    {
+        withdrawalTimeout = newWithdrawalTimeout;
+        emit WithdrawalTimeoutUpdated(withdrawalTimeout);
+        newWithdrawalTimeout = 0;
+        withdrawalTimeoutChangeInitiated = 0;
+    }
+
+    function getRemainingWithdrawalDelayUpdateTime()
+        external
+        view
+        returns (uint256)
+    {
+        return
+            getRemainingChangeTime(
+                withdrawalDelayChangeInitiated,
+                withdrawalGovernanceDelay()
+            );
+    }
+
+    function getRemainingWithdrawalTimeoutUpdateTime()
+        external
+        view
+        returns (uint256)
+    {
+        return
+            getRemainingChangeTime(
+                withdrawalTimeoutChangeInitiated,
+                withdrawalGovernanceDelay()
+            );
+    }
+
+    /// @notice The time it takes to initiate and complete the withdrawal from
+    ///         the pool plus 2 days. This governance delay should be used for
```

Comment:
Maybe it's worth adding a note why exactly 2 days?

Reply:
What if we say "... plus 2 days to make a decision"? 

It's quite an arbitrary value, to be honest. I assumed 2 days are enough in most cases to be notified and make a decision, maybe it is not enough.

Reply:
Sure, fair enough.

Reply:
Actually, worth mentioning that every `finalize*` governance function is `onlyOwner`, so if the governance wants to give some extra time for underwriters they can do that at any moment. Those two days are an absolute minimum. I'll clarify it in the docs.

Reply:
Done as a part of https://github.com/keep-network/coverage-pools/pull/105/commits/f6c542d35278810ac95b149495b3edb6858cc149.